src/shared/components/Header/__tests__/__snapshots__/Header.test.tsx.snap

```diff
@@ -60,7 +60,11 @@ exports[`Header component Should render correctly 1`] = `
       mouseEnterDelay={0.15}
       mouseLeaveDelay={0.1}
       overlay={
-        <Menu>
+        <Menu
+          className=""
```

Comment:
since this `className` is empty, can it be removed?

Reply:
This was autogenerated from the snap